exporters/prometheus/config.go

```diff
@@ -57,3 +60,13 @@ func WithRegisterer(reg prometheus.Registerer) Option {
 		return cfg
 	})
 }
+
+// WithoutTargetInfo configures the Exporter to not export the resource `target_info` metric.
+// If not specified, the Exporter will create a `target_info` metric containing
```

Comment:
Go docs do not use markdown syntax.

```suggestion
// WithoutTargetInfo configures the Exporter to not export the resource target_info metric.
// If not specified, the Exporter will create a target_info metric containing
```